controllers/infrastructure/k8sinstallerconfig_controller.go

```diff
@@ -258,28 +258,6 @@ func (r *K8sInstallerConfigReconciler) ByoMachineToK8sInstallerConfigMapFunc(o c
 func (r *K8sInstallerConfigReconciler) reconcileDelete(ctx context.Context, scope *k8sInstallerConfigScope) (reconcile.Result, error) {
 	logger := scope.Logger
 	logger.Info("Deleting K8sInstallerConfig")
-
```

Comment:
Are we relying on the K8s garbage collection mechanism to clean this?

Reply:
Yup, we have owner reference set in the secret created by installer.